admin/class-admin-apple-news.php

```diff
@@ -135,13 +135,22 @@ public function __construct() {
 			add_action(
 				'rest_api_init',
 				function() {
-					register_rest_field(
-						'post',
-						'apple_news_notices',
-						[
-							'get_callback' => [ 'Admin_Apple_Notice', 'get_if_allowed' ],
-						]
-					);
+					$admin_settings = new Admin_Apple_Settings();
+					$settings       = $admin_settings->fetch_settings();
+					$post_types     = $settings->post_types;
```

Comment:
could simplify it a little and catch an error if one came up

```suggestion
					$post_types     = $admin_settings->fetch_settings()->post_types ?? [];
```